.yamllint.yaml

```diff
@@ -17,3 +17,5 @@ rules:
     check-keys: true  # by default, only values are checked
     quote-type: single
     required: only-when-needed
+  line-length:
+    ignore: .github/workflows/*.yaml
```

Comment:
This is a follow up of https://github.com/yaml/go-yaml/pull/163#discussion_r2482809948

While I understand it's easy to proceed this way, I'm a bit annoyed it disable the line length on everything, and not only the one containing hashes.

I would prefer to use the per line yamllint directive